src/tools/crawling_tools.py

```
@@ -170,6 +173,128 @@ async def scrape(ctx: Context, url: str) -> str:
+                            }
+                        )
+                        chunk_node.add_relationship("PART_OF", url)
+                        processed_content.structural_nodes.append(chunk_node)
+                    
+                    # Store in Neo4j
+                    if processed_content.structural_nodes:
+                        await neo4j_ingester.ingest(processed_content)
+                        neo4j_nodes_created = len(processed_content.structural_nodes)
+                        logger.info(f"✓ Stored {neo4j_nodes_created} structural nodes in Neo4j")
+                        
+                except Exception as neo4j_error:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)